addons/udes_delivery_control/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_stock_picking_extras,access_stock_picking_extras,model_stock_picking_extras,stock.group_stock_user,1,1,1,1
+access_stock_picking_vehicle_type_stock_user,access_stock_picking_vehicle_type_user,model_stock_picking_vehicle_type,stock.group_stock_user,1,0,0,0
+access_stock_picking_vehicle_type_manager,access_stock_picking_vehicle_type_manager,model_stock_picking_vehicle_type,stock.group_stock_manager,1,1,1,1
```

Comment:
I think there was a misunderstanding with this comment: https://github.com/unipartdigital/udes-open/pull/361#pullrequestreview-509158128

When I said Stock User for Picking's should be pulled across, I didn't mean instead of Stock Manager, I meant in addition to. :)

For Vehicle Type Stock User can view but not edit or delete, I think it should be able to do all of that. Whereas Stock Manager can do everything but does not have access to the Picking Extra model at all.

I think the permissions should look like this:

```
access_stock_picking_extras_stock_user,access_stock_picking_extras_stock_user,model_stock_picking_extras,stock.group_stock_user,1,1,1,1
access_stock_picking_extras_stock_manager,access_stock_picking_extras_stock_manager,model_stock_picking_extras,stock.group_stock_user,1,1,1,1
access_stock_picking_vehicle_type_stock_user,access_stock_picking_vehicle_type_user,model_stock_picking_vehicle_type,stock.group_stock_manager,1,1,1,1
access_stock_picking_vehicle_type_manager,access_stock_picking_vehicle_type_manager,model_stock_picking_vehicle_type,stock.group_stock_manager,1,1,1,1
```

Reply:
Ignore comment regarding Vehicle Type, original permissions were view only for Stock User as we only want Stock Manager to configure them. :)